### PR TITLE
Pull request for WAZO-1924-adhoc-conference-deletion

### PR DESCRIPTION
**Pull request by sduthil**
Depends-On: https://github.com/wazo-platform/wazo-calld-client/pull/32
Depends-On: wazo-platform/xivo-bus#25
Depends-On: https://github.com/wazo-platform/wazo-calld/pull/118

### CODE REVIEW

**Comment by sduthil** [comment]
recheck

**Comment by sduthil** [comment]
recheck

File: wazo_calld/plugins/adhoc_conferences/http.py [hunk 1]
```diff
@@ -28,3 +28,44 @@ def post(self):
...
         return adhoc_conference, 201
+
+
+class UserAdhocConferenceResource(AuthResource):
+
+    def __init__(self, adhoc_conference_service, auth_client):
+        self._adhoc_conference_service = adhoc_conference_service
+        self._auth_client = auth_client
+
+    @required_acl('calld.users.me.conferences.adhoc.create')
```
Comment (bloom1) [inline wazo_calld/plugins/adhoc_conferences/http.py:39]: Should it be `calld.users.me.conferences.adhoc.delete`?

Comment (pc-m) [reply wazo_calld/plugins/adhoc_conferences/http.py]: I fixed it to match the `api.yml` and rebased the branch on `master` to make Zuul faster when you guys get back.

Comment (sduthil) [reply wazo_calld/plugins/adhoc_conferences/http.py]: Thank you!

**Pull request merged**